pyfr/solvers/base/system.py

```diff
@@ -186,6 +187,9 @@ def _gen_kernels(self, eles, iint, mpiint, bcint):
     def rhs(self, t, uinbank, foutbank):
         pass
 
+    def compute_grads(self, t, uinbank):
+        pass
```

Comment:
Maybe have this raise a not implemented error and have the integrator catch this exception to decide when to fallback.